README.md

```diff
@@ -351,39 +356,18 @@ const myCustomHeaderFactory = () => withState(
   // actions
   {addTodo}
 )(CustomHeader);
-
 ```
 
-#### 5. How to add data to map
-In order to interact with a kepler.gl instance and add new data to it the following methods are available:
-- updateVisData
-- addDataToMap
-It is also important to remember that Kepler.gl provides an easy API (```KeplerGlSchema.getConfigToSave```) to generate a dump of the current kepler instance configuration.
+### 5. How to add data to map
+To interact with a kepler.gl instance and add new data to it, you can dispatch __`addDataToMap`__ action from anywhere inside your app. It adds dataset to kepler.gl instance and update the full configuration (mapState, mapStyle, visState).
 
-##### addDataToMap
-This method is similar to UpdateVisData but it is able to update the full kepler.gl configuration (mapState, mapStyle, visState).
-This action takes an object as input with the following properties:
-```js
-{
-    datasets | object: same as UpdateVisData
-    options | object: same as UpdateVisData
-    config | object: this object will contain the full kepler.gl instance configuration {mapState, mapStyle, visState}.
-}
-```
+Kepler.gl provides an easy API `KeplerGlSchema.getConfigToSave` to generate a json blob of the current kepler instance configuration.
 
-<p id="config-notice"> It is important to notice that the config object value will always have higher precedence than the options properties.
-For instance, if you provide <code>{centerMap: true}</code> as part of the options object and in your config object you pass
-the mapState configuration with latitude and longitude define as it follows:</p>
+The options properties will alway have higher precedence than config object value.
+For instance, if you provide `{centerMap: true}` as part of the options object and in your config object you pass
```

Comment:
Can you double check this? if i recall correctly this was the issue we were talking about were the config override centerMap: true

Reply:
I thought Charlie's diff didn't land

Reply:
actually, you are right, the config still take precedence